hugo/content/faq/volunteer.md

```diff
@@ -3,6 +3,8 @@ Title: How can I help improve the ACL Anthology?
 weight: 4
 ---
 
-The best way is to browse our [Github issues](https://twitter.com/TheGarlandSitIn/status/1123628614342385666) page, and see what needs to be done that might overlap with your interests or skills.
+We welcome your contributions!
+In fact, since the Anthology is run entirely on volunteer energy, we need them.
+The best way to contribute is to browse our [Github issues](https://twitter.com/TheGarlandSitIn/status/1123628614342385666) page, and see what needs to be done that might overlap with your interests or skills.
```

Comment:
JHU sit-in twitter message linked instead of the github issues?!  Interesting that nobody caught that before.

Reply:
Oh boy.